android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.translate
+
+/**
+ * The operation the translations engine is performing.
+ *
+ * @see TRANSLATE - The page should be translated.
+ * @see RESTORE - A translated page should be restored.
```

Comment:
Could we move these parameter descriptions next to their enum? See https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt#595

Reply:
Thanks for the link!